apps/web/src/features/safenet/components/SafenetBalanceOverview/index.tsx

```diff
@@ -0,0 +1,78 @@
+import ChainIndicator from '@/components/common/ChainIndicator'
+import FiatValue from '@/components/common/FiatValue'
+import useBalances from '@/hooks/useBalances'
+import { useDarkMode } from '@/hooks/useDarkMode'
+import InfoIcon from '@/public/images/notifications/info.svg'
+import SafenetDarkIcon from '@/public/images/safenet/safenet-icon-dark.svg'
+import SafenetIcon from '@/public/images/safenet/safenet-icon.svg'
+import type { SafenetBalance } from '@/utils/safenet'
+import { Box, Divider, SvgIcon, Tooltip, Typography } from '@mui/material'
+import { useMemo } from 'react'
+import css from './styles.module.css'
+
+function sumBalancesByChainId(balances: SafenetBalance[]): { [chainId: string]: number } {
+  return balances.reduce(
+    (acc, { chainId, fiatBalance }) => {
+      acc[chainId] = (acc[chainId] || 0) + parseFloat(fiatBalance)
+      return acc
+    },
+    {} as { [chainId: string]: number },
+  )
+}
+
+const SafenetBalanceOverview = () => {
+  const { balances } = useBalances()
+  const isDarkMode = useDarkMode()
+
+  const totalSafenetValueFiat = useMemo(() => {
```

Comment:
I do not think we can actually sum up fiat balances here.
If we do want this, we need to consider the correct fiat conversion rates.

I would suggest to only go for USDC balances for now until we support multiple tokens.